website/docs/deploy/securing-unleash-v3.md

```diff
@@ -25,7 +25,7 @@ unleash
     adminAuthentication: 'custom',
     preRouterHook: myCustomAdminAuth,
   })
-  .then(unleash => {
+  .then((unleash) => {
```

Comment:
Well, yes and no. We _do _ generally try and follow prettier's formatting for our files now (including the source code), so it _is_ the right thing to do. That said, I wasn't expecting it to happen, but I'm happy to leave it in because it conforms to our current standard.

Reply:
I'm ok either way, I usually follow a different coding standard removing unnecessary parenthesis, but if this is the standard we're using in our code base I'm fine with it.

Reply:
Yeah, I agree, but we use prettier for our code base and for our docs, so it is what it is, I suppose 💁🏼